lib/os.h

```diff
@@ -20,6 +20,9 @@
 #include "config.h"
 #endif
 
+#ifdef _WIN32
```

Comment:
Is `_WIN32` what we want here, or would _MSC_VER be better? mingw probably acts like gcc?

cc @evpobr

Reply:
For _MSC_VER: definitely needed. Harmless for others.

Reply:
Right, although the documented intention is slightly different.